src/DynamicRupture/Initializers/RateAndStateInitializer.cpp

```diff
@@ -189,16 +71,14 @@ std::pair<real, real>
                                                             real initialSlipRate) {
   real absoluteTraction = std::sqrt(std::pow(tractionXY, 2) + std::pow(tractionXZ, 2));
   real tmp = std::abs(absoluteTraction / (rs_a * pressure));
-  real stateVariable = rs_a / rs_sr0 *
+  real stateVariable = rs_sl0 / rs_sr0 *
```

Comment:
seems wrong

Reply:
Hi, I made the error myself earlier. IMO it is correct now.
For the slip and aging law (l74), this should be the initial state variable: https://github.com/SeisSol/SeisSol/blob/bc4b4625fb1be72ccd370a30aa6e3c3351e006ed/src/Physics/ini_model_DR.f90#L676
For FastVelocityWeakening (l106) this should be the correct initial state variable: https://github.com/SeisSol/SeisSol/blob/bc4b4625fb1be72ccd370a30aa6e3c3351e006ed/src/Physics/ini_model_DR.f90#L673

Reply:
ok, sorry, I just thought you may have changed the variable name by mistake.
Yes that's correct.

Reply:
Always good to have a second thought!